### DISCUSSION TITLE
Customize pagination bar

### DISCUSSION THREAD

**Discussion by luizfilho0205** [orchidsoftware/platform#2671] [Q&A]
How can I customize colors or text decoration in the pagination bar on Table?
My target is the ".pagination .page-item.active" CSS class to more expressively indicate to the user which page is active.

This is how it is currently:

![pagination_bar_standard](https://github.com/orchidsoftware/platform/assets/83550688/cbaa818c-d282-4ef1-b4bd-c1c7c8b232fb)

I would like it to look like this:

![pagination_bar_underline](https://github.com/orchidsoftware/platform/assets/83550688/3a6c5bf9-73af-4639-9645-fdb7fcac53ce)

Thanks for your time.

**Comment by c-gross** [comment]
Easier is to add some custom CSS for .page-item.active
But you can also overwrite the vendor/orchid/platform/resources/views/partials/pagination.blade.php template and add your own css classes.
